### PR TITLE
Create 0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json

### PR DESCRIPTION
**Pull request by tmyoungjr**
add Moonlite (MNL) to tokens

### CODE REVIEW

File: tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "instagram": "",
+    "linkedin": "",
+    "reddit": "",
+    "slack": "",
+    "telegram": "",
+    "twitter": "",
+    "whitepaper": "",
+    "youtube": ""
+  },
+  "pricing": {
```
Comment (ligi) [inline tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json:35]: these fields are not supported - also empty - please rename or elaborate why they are there

**Review by ligi — CHANGES_REQUESTED** [review]

File: tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json (hunk 1 above)
Comment (tmyoungjr) [reply tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json]: Should be cleaner

File: tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "symbol": "MNL",
+  "address": "0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402",
+  "decimals": 18,
+  "name": "Moonlite",
+  "type": "ERC20",
+  "website": "https://www.moonlite.io/",
```
Comment (ligi) [inline tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json:7]: this "," makes it not conform to json

**Review by ligi — CHANGES_REQUESTED** [review]
remove the , to make it JSON conform

File: tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json (hunk 2 above)
Comment (tmyoungjr) [reply tokens/eth/0x0d62dc6Cd8C81dCA8cAADcCF01FfE7C1F31D9402.json]: resolved JSON error

**Review by ligi — APPROVED** [review]

**Pull request merged**